src/utils/pause/frames/getLibraryFromUrl.js

```diff
@@ -78,7 +78,13 @@ const libraryMap = [
   },
   {
     label: "Angular",
-    pattern: /angular/i,
+    pattern: new RegExp(
+      "angular\\.|" +
+        "angular.*" +
+        "(animations|common|core|elements|forms|platform-browser|" +
```

Comment:
I'm slightly worried that using these specific strings will cause a maintenance issue, in that every time they rename a file or add one, we'll need to make updates.  Can we do a `angular.([A-z+])` type of regex?  It's more generic but I doubt many people purposely prefix the specific JS with `angular.`

Reply:
Good call - switched this out so that it's much simpler :)